src/classes/services/CaseService.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Mariya on 25.01.2023.
+ */
+
+public inherited sharing class CaseService {
+
+    public static List<Case> getCasesWithNewOwnerId(List<Case> newCases, Map<Id, Case> oldCasesByIds) {
```

Comment:
MINOR
Better to set `private` access modificator as it is not used anywhere outside.